RNMixpanel/RNMixpanel.m

```diff
@@ -23,8 +23,7 @@ @implementation RNMixpanel
 
 // sharedInstanceWithToken
 RCT_EXPORT_METHOD(sharedInstanceWithToken:(NSString *)apiToken) {
-    [Mixpanel sharedInstanceWithToken:apiToken];
```

Comment:
In Mixpanel's iOS library, [```sharedInstaneWithToken```](https://github.com/mixpanel/mixpanel-iphone/blob/master/Mixpanel/Mixpanel.m#L38) creates an instance, stores it in a dictionary of ```instances```, and returns the instance.

Their [````sharedInstance``` method](https://github.com/mixpanel/mixpanel-iphone/blob/master/Mixpanel/Mixpanel.m#L58) just returns the instance that was created first.